### ISSUE TITLE
API Tracking issue: search

### ISSUE THREAD

**Issue by bibliotechie** [bibliotechie/antithesis#9]
This is a tracking issue for the Hypothesis API endpoint `GET /search`. Corresponding wiki page: https://github.com/bibliotechie/antithesis/wiki/Hypothesis-API%3A-search

**Comment by bibliotechie** [comment]
We don't need full search functionality to start off with, but because this is used to retrieve annotations for a page, etc., some subset is needed for MVP. I'm going to break that off into its own issue.

**Comment by bibliotechie** [comment]
Depends on #32.
